templates/bluemix/manifest.yml

```diff
@@ -0,0 +1,16 @@
+declared-services:
+  cloudant-db:
+    label: cloudantNoSQLDB
+    plan: Lite
+applications:
+- path: .
+  memory: 256M
```

Comment:
After detecting `--bluemix`, we should prompt the users for values for all overridable Bluemix defaults.

@seabaylea you might want to join us in the discussion.